lighthouse/classes/services/mongo.py

```diff
@@ -38,3 +38,23 @@ def get_samples_from_mongo_for_barcode(self, barcode):
         samples = list(samples_collection.find({FIELD_PLATE_BARCODE: barcode}))
 
         return samples
+
+    def get_source_plates_from_barcodes(self, barcodes):
+        with app.app_context():
+            source_plates_collection = app.data.driver.db.source_plates  # type: ignore
```

Comment:
You can cast the type here to `Eve` I think.

Reply:
Done